pkg/util/types/pvc.go

```diff
@@ -76,3 +76,37 @@ func IsSharedPVCFromClient(client kubecli.KubevirtClient, namespace string, clai
 	}
 	return
 }
+
+// GetPVCHostPathFromStore determines if the persistent volume bound to the passed in claim is a host path based
+// volume, and if so, returns the path of the volume. Returns a blank path if not a hostpath volume
+func GetPVCHostPathFromStore(pvcStore cache.Store, pvStore cache.Store, namespace string, claimName string) (string, error) {
+	var pvc *k8sv1.PersistentVolumeClaim
+	obj, exists, err := pvcStore.GetByKey(namespace + "/" + claimName)
+	if err != nil {
+		return "", err
+	}
+	if !exists {
+		return "", fmt.Errorf("Unable to find PVC %s/%s", namespace, claimName)
+	}
+	if _, ok := obj.(*k8sv1.PersistentVolumeClaim); ok {
+		pvc = obj.(*k8sv1.PersistentVolumeClaim)
+	}
+
+	if pvc.Status.Phase == k8sv1.ClaimBound && pvc.Spec.VolumeName != "" {
+		var pv *k8sv1.PersistentVolume
+		obj, exists, err := pvStore.GetByKey(pvc.Spec.VolumeName)
+		if err != nil {
+			return "", err
+		}
+		if !exists {
+			return "", fmt.Errorf("Unable to find PV %s", pvc.Spec.VolumeName)
+		}
+		if _, ok := obj.(*k8sv1.PersistentVolume); ok {
+			pv = obj.(*k8sv1.PersistentVolume)
+		}
+		if pv.Spec.HostPath != nil {
+			return pv.Spec.HostPath.Path, nil
+		}
```

Comment:
One thing that's unclear to me is whether the hostpath's path is actually sensitive information. Are we certain this information is sensitive? If it's not sensitive, we have other options here. We could even pass the path as an annotation on the VMI.

Reply:
The only place this is available is in the PV, and PVs are not normally readable by regular users, therefore it is sensitive information.

Reply:
> The only place this is available is in the PV, and PVs are not normally readable by regular users, therefore it is sensitive information.

secrets in the user's namespace probably are though, which is where we're placing the information. 

if we feel strongly that this is sensitive info, maybe we should consider only placing this information in a secret in the kubevirt install namespace

Reply:
But how would we pass the secret to the pod? The secret is created and owned by the kubevirt controller sa, regular users should not be able to read them.